CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ find_package(catkin REQUIRED
     moveit_ros_planning_interface
     pluginlib
     geometric_shapes
+    moveit_visual_tools
```

Comment:
> But I've poured many years of work into those two packages though and kind of like to keep it under my name.

I believe this is a perspective you know very well from other programmers and you argued against it quite a lot in recent days. I don't see how you don't "keep it under your name" if we move it to ros-planning. You're the listed author and maintainer.
It just means that the ros-planning group officially supports/endorses the package and it gets a more "official" location.
I agree that we might not want to merge it into the `moveit` repository.
Maybe it's worth a thought to add it to the `moveit` meta-package.
After all, this patched tutorial is a lot of advertisement for your package and makes it look like an off-the-shelf moveit component imho. You don't get it on your machine if you install "moveit" (e.g. `ros-kinetic-moveit`) though.

You say it's a powerful and useful package that supports the MoveIt framework. I believe we should try to keep such packages under the ros-planning umbrella where possible.

> Otherwise I'd be more restricted in developing it further - its kind of my favorite pet project repo.

If you don't want to keep the API you use in the tutorials *stable*, I'm definitely against using it in the tutorials at all! But I suppose you do.
If this just refers to merging it into the repo, see above.